spec/system/manage_maps_spec.rb

```diff
@@ -16,6 +16,11 @@
     attach_file "Image", file_fixture("dwarven-excavation.jpg")
     click_on "Create Map"
     expect(page).to have_content "Dwarven Excavation"
+    created_map = campaign.maps.first
+    expect(created_map.height).to be_present
+    expect(created_map.width).to be_present
+    expect(created_map.x).to eq(created_map.width / 2)
+    expect(created_map.y).to eq(created_map.height / 2)
```

Comment:
Could you test the map's attributes by finding the node, rather than loading from the DB? You will need to select the map first I think.

That way we could also also have tests for the panning and zooming across sessions like the map switching at the bottom of this file.

Reply:
Done in 2a0b65c